### PR TITLE
Publish stable version of stream_transform

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
CC @lukaszkolodziejczyk

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## 2.0.0-nullsafety.0
+## 2.0.0
```
Comment (natebosch) [inline CHANGELOG.md:1]: We have been keeping the `-nullsafety.0` changelog entries in place and adding new ones for the stable release.

Should we be changing our approach? cc @jakemac53

Comment (jakemac53) [reply CHANGELOG.md:1]: I don't know... I can see an argument either way.

In the case of breaking changes, they can be a bit buried with the current approach.

Comment (kevmoo) [reply CHANGELOG.md:1]: My thoughts exactly. The null safe versions are blips in history. Noise for the vast majority of users.

**Review by natebosch — APPROVED** [review]

**Pull request merged**